lib/main_common.pm

```diff
@@ -534,10 +545,10 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    # SYSTEM_ROLE_STYLE will be included in Mediums for openSUSE products progressively offering this new dialog
+    # i.e.: not all staging will receive it at the same time
+    loadtest((get_var('SYSTEM_ROLE_STYLE') || is_caasp('kubic')) ? "installation/system_role" : "installation/installer_desktopselection") if (is_opensuse);
```

Comment:
It's pretty hard to read, could you please put in multiple line if-else? I know, it's matter of taste, but this `if is_opensuse` in the end of the line is kind of hidden.

Reply:
the if at the end I didn't like neither :) , probably you're right due to strings for the test cases are also pretty long.